### PR TITLE
[Havoc] added in t21 2pc

### PR DESCRIPTION
**Pull request by CollCrom**
image

### CODE REVIEW

File: src/Parser/DemonHunter/Havoc/CHANGELOG.js [hunk 1]
```diff
@@ -4,6 +4,16 @@ import { Hewhosmites, Mamtooth } from 'MAINTAINERS';
```
Comment (sMteX) [inline src/Parser/DemonHunter/Havoc/CHANGELOG.js]: Missing imports for `SpellLink, SPELLS, ItemLink` and `ITEMS` causing Travis to fail.

File: src/Parser/DemonHunter/Havoc/CHANGELOG.js [hunk 2]
```diff
@@ -4,6 +4,16 @@ import { Hewhosmites, Mamtooth } from 'MAINTAINERS';
 
 
 export default [
+	{
+		date: new Date('2018-1-26'),
```
Comment (sMteX) [inline src/Parser/DemonHunter/Havoc/CHANGELOG.js]: I think the date has to be in yyyy-mm-dd format, `2018-01-26`

**Comment by kfinch** [comment]
A recent changelogs update caused a merge conflict too, you'll need to resolve that before merging.

**Comment by CollCrom** [comment]
I fixed all my changelog dates since I had the wrong format for all of them as well

**Pull request merged**